notifier/stomp/deliverer.go

```diff
@@ -0,0 +1,62 @@
+package stomp
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"path"
+
+	gostomp "github.com/go-stomp/stomp"
+	"github.com/google/uuid"
+	clairerror "github.com/quay/clair/v4/clair-error"
+	"github.com/quay/clair/v4/notifier"
+)
+
+// Deliverer is a STOMP deliverer which publishes a notifier.Callback to the
+// the broker.
+type Deliverer struct {
+	conf Config
+	fo   *FailOver
+}
+
+func New(conf Config, fo *FailOver) (*Deliverer, error) {
+	var c Config
+	var err error
+	if c, err = conf.Validate(); err != nil {
+		return nil, err
+	}
+	return &Deliverer{
```

Comment:
Configuration is stored twice in Deliver and in Failover and both have separate TLS configurations. Why is that?

Reply:
They both use the same config. Validate() returns a copy of a Config with all private members parsed